model-optimizer/extensions/middle/ONNXResize11ToInterpolate.py

```diff
@@ -96,7 +97,7 @@ def replace_resize(graph: Graph, resize: Node):
 
     dst_dtype = np.float32  # even if data_type=FP16 use float32 for shape values
 
-    if num_of_inputs == 3:
+    if not resize.is_in_port_connected(3):
```

Comment:
The same comment as above. For more backward compatibility we should write the following code:
```python
if resize.is_in_port_connected(2):
```